torch/testing/_internal/common_cuda.py

```diff
@@ -15,6 +15,10 @@
 TEST_CUDNN = TEST_CUDA and torch.backends.cudnn.is_acceptable(torch.tensor(1., device=CUDA_DEVICE))
 TEST_CUDNN_VERSION = torch.backends.cudnn.version() if TEST_CUDNN else 0
 
+CUDA11 = torch.version.cuda and float(torch.version.cuda) >= 11
```

Comment:
These are confusing macros - CUDA11 and SM53 use >= comparisons, and CUDA9 uses `==`

Reply:
The CUDA11 and SM53 variables need names like "AtLeastCUDA11" and "AtLeastSM53" or "CUDA11OrLater" and "SM53OrLater" to be clear.

Reply:
fixed